src/spec/delivery_service.md

```diff
@@ -99,8 +101,9 @@ To distribute MLS messages the DS sends messages on to the local QS to enqueue i
 
 In cases where the QS responds with a message indicating that a target queue doesn't exist, the DS assumes that the queue was deleted and proposes the removal of the corresponding group member as specified [here](delivery_service.md#ds-induced-removals).
 
-## Activity time
+## Activity times
 
+The DS tracks both the last time individual group members were active (encrypted at rest as part of the group state), as well as the last time the group in general was actively used. A timestamp consists only of the month and the year of last activity.
```

Comment:
Currently, the [AS State](https://docs.phnx.im/spec/authentication_service.html#as-state) has a reference to timestamps, too. It could be worth mentioning the granularity of the stored timestamp there, too.